starters/e2e/e2e.attributes.spec.ts

```diff
@@ -62,12 +62,12 @@ test.describe("attributes", () => {
       await stuffBtn.click();
       await expect(inputCopy).toHaveJSProperty("value", "Hello");
       await expect(inputValue).toHaveText("Hello");
-      await expect(renders).toHaveText("2");
+      await expect(renders).toHaveText("1");
```

Comment:
Why the change here?

Reply:
this additional rerender was caused by removed console.log. I don't know if this is needed